src/main/java/TaskList.java

```diff
@@ -14,34 +23,60 @@ public void printList(){
         System.out.print("____________________________________________________________\n");
     }
 
-    public void markTaskDone(int index){
+    /**
+     * This method takes in an index and marks the task in the ArrayList
+     * with the corresponding index as 'done'
+     * @param index Takes in an integer that and marks the corresponding task in
+     *              the TaskList as 'done'.
+     * */
+    public void markTaskDone(int index) {
         tasks.get(index).markAsDone();
         System.out.println("____________________________________________________________\n"
                 + "Nice! I've marked this task as done:\n" + tasks.get(index).toString()
                 + "\n____________________________________________________________\n");
     }
 
```

Comment:
Perhaps you could add in a UI class in order to keep your code simpler. Just a simple clean up which can be done near the end which could really help others understand your code better!